build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
   compile 'com.fasterxml.jackson.core:jackson-core:2.9.5'
   compile 'com.fasterxml.jackson.core:jackson-annotations:2.9.5'
   compile 'com.fasterxml.jackson.core:jackson-databind:2.9.5'
+  compile 'com.sun.mail:javax.mail:1.6.0'
```

Comment:
Why this library over http://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/EmailValidator.html? Reference: https://stackoverflow.com/questions/624581/what-is-the-best-java-email-address-validation-method